src/XCCDF/result.c

```diff
@@ -178,8 +178,14 @@ void xccdf_result_fill_sysinfo(struct xccdf_result *result)
 		return;
 
 	_xccdf_result_clear_metadata(XITEM(result));
+
+	/* override target name by environment variable */
+	const char *target_hostname = getenv("OSCAP_PROBE_PRIMARY_HOST_NAME");
```

Comment:
The name of the environment variable is not very self-documenting. It's not related to OVAL probes and it is not primary.

Reply:
@mpreisler Yes, I agree, but the variable is already set in oscap-chroot in oscap-vm. Fortunately it's not read anywhere. Do you think I can rename it and rename all ocurences? Isn't it used somewhere outside of OpenSCAP?

Reply:
I don't think it's used outside of OpenSCAP. I think we should rename it before people start using it.